src/ClientData/ModuleData.cpp

```diff
@@ -173,4 +169,8 @@ std::vector<const FunctionInfo*> ModuleData::GetFunctions() const {
   return result;
 }
 
+std::string ModuleData::GetLoadedModuleNameByPath(std::string_view module_path) {
```

Comment:
@beckerhe I also moved this static function from the former FunctionUtils to ModuleData.